components/src/web-components/visualization/gs-mutations-over-time.tsx

```diff
@@ -146,6 +154,7 @@ export class MutationsOverTimeComponent extends PreactLitAdapterWithGridJsStyles
                     lapisDateField={this.lapisDateField}
                     displayMutations={this.displayMutations}
                     initialMeanProportionInterval={this.initialMeanProportionInterval}
+                    initialHideGaps={this.initialHideGaps}
```

Comment:
This isn't only initial anymore, let's rename?
```suggestion
                    hideGaps={this.hideGaps}
```